dumbTimeStamp.c

```diff
@@ -0,0 +1,55 @@
+// dumbTimeStamp.c for wSQL
+
+// This creates a timestamp from the system time. Looking at time.h for a better alternative
+
+// includes
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+
+// defines
+#define     BUFSIZE    200
+
+// structs
+struct timeval tv;
+
+// prototypes
+
+
+int main (void) {
```

Comment:
I don't know the correct library for getting the time in the format we need so I made this as a placeholder until I get that squared away. As far as I can tell sys/time.h only returns the number of seconds since 1970 yet this is the library Abbassi suggested I use